src/ts/entitysystem/components/drawing/ImageDrawable.ts

```diff
@@ -0,0 +1,190 @@
+///<reference path="./Drawable.ts"/>
+///<reference path="../../../util/JsonLoader.ts"/>
+module entityframework.components.drawing {
+
+    import observe = framework.observe;
+    import serialize = util.serialize;
+
+    /**
+     * Represents a drawable that can draw an image.
+     */
+    @serialize.ProvideClass(ImageDrawable, "ild::ImageDrawable")
+    export class ImageDrawable extends Drawable {
```

Comment:
I think the code should be reworked to remove _imageFile and _image from the drawable.  They are not a core part of the model.